test/development/app-dir/show-error-when-missing-html-or-body-in-root-layout/show-error-when-missing-html-or-body-in-root-layout.test.ts

```diff
@@ -0,0 +1,39 @@
+import { nextTestSetup } from 'e2e-utils'
+import { getRedboxHeader, hasRedbox } from 'next-test-utils'
+
+describe('show error when missing html or body in root layout', () => {
+  const { next } = nextTestSetup({ files: __dirname })
+
+  it('should show error overlay', async () => {
+    const browser = await next.browser('/')
+
+    expect(await hasRedbox(browser)).toBe(true)
+    expect(await getRedboxHeader(browser)).toMatchInlineSnapshot(`
```

Comment:
We can use `getRedboxDescription` to only get the description part.

Was having a lot of annoying matching with the header in hmr test before, since this irrelavent part is also needed to be updated again if we updated the header 
```
"1 of 1 unhandled error
Unhandled Runtime Error
```